Sources/macro.js

```diff
@@ -1,6 +1,9 @@
+import globalThisShim from 'globalthis';
 import vtk from './vtk';
 import ClassHierarchy from './Common/Core/ClassHierarchy';
 
+const vtkGlobal = globalThisShim(); // returns native globalThis if compliant
```

Comment:
can you `export` it so it could be used outside?

Reply:
Sure

Reply:
I had to export from `vtk.js` becase `macro.js` depends on it.